apps/notifications/src/standalone/offline-wrapper.jsx

```diff
@@ -1,33 +0,0 @@
-import React from 'react';
```

Comment:
I'm assuming we're deleting this because it's unused?

I'm actually not sure it was ever used. It was added in #26757 but I don't see why - I can't find it in the old notifications panel repos at all.

Perhaps @dmsnell might know?

Reply:
Yep, forgot to add that comment. 

I can't find where this file is used. I did found the first reference to the file in the original `notifications-panel` repo in https://github.com/Automattic/notifications-panel/commit/98915a37ecd947cfc265a29fd0e0f3ebc0d15b75#diff-04d29094f91ef9ef93b8f5f30d52fa4e, but still the file itself doesn't exist in that commit.